seed/challenges/01-front-end-development-certification/basic-javascript.json

```diff
@@ -3458,6 +3456,9 @@
         "assert(code.match(/break/g).length > 2, 'message: You should have at least 3 <code>break</code> statements');"
       ],
       "type": "waypoint",
+      "MDNlinks": [
```

Comment:
As a note, I had left it cause it didn't seem to effect anything else, just the position in the file.  When I worked on the pull request I submitted after this one though #7863, the entire file had been re-ordered so I went back and manually changed that one, since the diff would have been the entire file.